### PR TITLE
Update leverage-containers-orchestrators.md

### PR DESCRIPTION
**Pull request by karoldeland**
No downtime deploy is achievable when using some mecanisms like Azure App Service Slots.

## Summary

Describe your changes here.

Being more realist about the consequences of monolith app deployment.

### CODE REVIEW

**Comment by sughosneo** [comment]
Hi @karoldeland, thank you for submitting this PR.

File: docs/architecture/cloud-native/leverage-containers-orchestrators.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Although they have the benefit of simplicity, monolithic architectures face a nu
 
 ### Deployment
 
-Monolithic applications require a full deployment of the entire application, even if only a small change has been made. Full deployments can be expensive and error prone. Additionally, they require a restart of the application, which temporarily impacts unavailability.
+Monolithic applications require a full deployment of the entire application, even if only a small change has been made. Full deployments can be expensive and error prone. Additionally, they require a restart of the application, which may temporarily impacts unavailability if not managed accordingly while deploying.
```
Comment (sughosneo) [inline docs/architecture/cloud-native/leverage-containers-orchestrators.md:23]: 1. `error prone` could be written as `error-prone`
2. Rephrased sentence - 
`Additionally, they require a restart of the application, which may temporarily impact unavailability if not managed accordingly.`

CC: @nishanil

Comment (nishanil) [inline docs/architecture/cloud-native/leverage-containers-orchestrators.md:23]:
```suggestion
Additionally, they require a restart of the application, which may temporarily impact availability if zero-downtime techniques are not applied while deploying. 
```

Comment (karoldeland) [reply docs/architecture/cloud-native/leverage-containers-orchestrators.md]: Used @nishanil's version.

**Review by nishanil — APPROVED** [review]

**Pull request merged**